### DISCUSSION TITLE
DATAGRID ColumnSizeMode

### DISCUSSION THREAD

**Discussion by jorge-teixeira** [bitfoundation/bitplatform#5660] [General]
Is there a way to format table or data grid columns based on the column content? For example, if a column contains a very large text description, that column could be wider and larger than other columns. However, I'd like to control the maximum length that column can have, so that it doesn't become too wide in the table. Is there a way to configure this, so that columns with more content can be wider up to a certain maximum point?

**Comment by msynk** [comment]
I think what you need can be solved by using the `BitDataGridTemplateColumn` component.
